doc/manual/install-judgehost.rst

```diff
@@ -31,7 +31,7 @@ Software requirements
 For Debian::
 
   sudo apt install make pkg-config sudo debootstrap libcgroup-dev \
-        php-cli php-curl php-json php-xml php-zip lsof procps
+        php-cli php-curl php-json php-xml php-zip lsof procps gcc g++
```

Comment:
It is true that you need a C/C++ compiler, but that could also be `clang`. But maybe mention that as a comment below this.

Reply:
The Red Hat packages should then also be updated below.